apps/api/src/app/inbox/e2e/get-preferences.e2e.ts

```diff
@@ -9,22 +10,28 @@ describe('Get all preferences - /inbox/preferences (GET)', function () {
     await session.initialize();
   });
 
-  it('should always get the global preferences even if workflow preferences are not present', async function () {
+  it('should return no global preferences if workflow preferences are not present', async function () {
     const response = await session.testAgent
       .get('/v1/inbox/preferences')
       .set('Authorization', `Bearer ${session.subscriberToken}`);
 
     const globalPreference = response.body.data[0];
 
-    expect(globalPreference.channels.email).to.equal(true);
-    expect(globalPreference.channels.in_app).to.equal(true);
+    expect(globalPreference.channels.email).to.equal(undefined);
+    expect(globalPreference.channels.in_app).to.equal(undefined);
     expect(globalPreference.level).to.equal('global');
     expect(response.body.data.length).to.equal(1);
   });
 
-  it('should get both global and workflow preferences if workflow is present', async function () {
+  it('should get both global preferences for active channels and workflow preferences if workflow is present', async function () {
     await session.createTemplate({
       noFeedId: true,
+      steps: [
+        {
+          type: StepTypeEnum.EMAIL,
+          content: 'Test notification content',
+        },
+      ],
```

Comment:
Explicitly specifying that only email is present, to assert that undefined should not appear.